pkgs/data/themes/pitch-black/default.nix

```
@@ -0,0 +1,30 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+
+stdenvNoCC.mkDerivation rec{
+  pname = "pitch-black";
+  version = "unstable-2019-07-23";
+
+  src = fetchFromGitHub {
+    repo = pname;
+    owner = "freefreeno";
+    rev = "d8039341419aef1157c030bf3d9237bd926e0b95";
+    hash = "sha256-Rn3ZMBD6srIkYFNN3HT5JFP46Akodmeqz5tbV2/2ZDA=";
+  };
+
+  dontBuild = true;
+
+  installPhase = ''
+    rm LICENSE README.md
+    mkdir -p $out/share
+    mv GTK $out/share/themes
+    mv * $out/share
+  '';
+
+  meta = with lib; {
+    description = "Pitch-Black theme is a dark plasma theme built with usability in mind and it looks great.";
```

There shouldn't be a dot at the end of this, see [contributing.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#submitting-changes).
```suggestion
    description = "Pitch-Black is a dark plasma theme built with usability in mind";
```
The last part also seems a bit subjective